pkgs/development/compilers/llvm/default.nix

```diff
@@ -25,7 +25,7 @@ let
     "18.1.8".officialRelease.sha256 = "sha256-iiZKMRo/WxJaBXct9GdAcAT3cz9d9pnAcO1mmR6oPNE=";
     "19.1.7".officialRelease.sha256 = "sha256-cZAB5vZjeTsXt9QHbP5xluWNQnAHByHtHnAhVDV0E6I=";
     "20.1.8".officialRelease.sha256 = "sha256-ysyB/EYxi2qE9fD5x/F2zI4vjn8UDoo1Z9ukiIrjFGw=";
-    "21.1.2".officialRelease.sha256 = "sha256-SgZdBL0ivfv6/4EqmPQ+I57qT2t6i/rqnm20+T1BsFY=";
+    "21.1.7".officialRelease.sha256 = "sha256-nuFnzfj2tSIdawLb6aZkz4xv7nD6sHGqpKOInHwmUlg=";
```

Comment:
When building this change, this hash was not working, the correct one being `sha256-SaRJ7+iZMhhBdcUDuJpMAY4REQVhrvYMqI2aq3Kz08o=` instead

Reply:
> that's weird:
> 
> ```
> nix hash file /tmp/llvmorg-21.1.7.tar.gz
> sha256-nuFnzfj2tSIdawLb6aZkz4xv7nD6sHGqpKOInHwmUlg=
> ```

The hash computation is done on the extracted directory, not the compressed archive